packages/blade/src/components/Typography/Heading/__tests__/Heading.web.test.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import renderWithTheme from '../../../../_helpers/testing/renderWithTheme.web';
+import Heading from '../';
+
+beforeAll(() => jest.spyOn(console, 'error').mockImplementation());
+afterAll(() => jest.restoreAllMocks());
+
+describe('<Heading />', () => {
+  it('should render Heading with variant "small"', () => {
+    const displayText = 'Get Started With Payment Gateway';
+    const { container, getByRole, getByText } = renderWithTheme(
+      <Heading type="normal" variant="small" weight="regular">
+        {displayText}
+      </Heading>,
+    );
+    expect(getByRole('heading', { level: 6 })).toBeInTheDocument();
+    expect(getByText('Get Started With Payment Gateway')).toBeInTheDocument();
+    expect(container).toMatchSnapshot();
+  });
+
+  it('should render Heading with variant "medium"', () => {
+    const displayText = 'Get Started With Payment Gateway';
+    const { container, getByRole, getByText } = renderWithTheme(
+      <Heading type="muted" variant="medium" weight="regular">
+        {displayText}
+      </Heading>,
+    );
+    expect(getByRole('heading', { level: 5 })).toBeInTheDocument();
+    expect(getByText('Get Started With Payment Gateway')).toBeInTheDocument();
+    expect(container).toMatchSnapshot();
+  });
+
+  it('should render Heading with variant "large"', () => {
+    const displayText = 'Get Started With Payment Gateway';
+    const { container, getByRole, getByText } = renderWithTheme(
+      <Heading type="subdued" variant="large" weight="regular">
+        {displayText}
+      </Heading>,
+    );
+    expect(getByRole('heading', { level: 4 })).toBeInTheDocument();
+    expect(getByText('Get Started With Payment Gateway')).toBeInTheDocument();
+    expect(container).toMatchSnapshot();
+  });
+
+  it('should throw error when variant is "subheading" but weight "regular" is passed', () => {
```

Comment:
nit: Maybe we should also have a test for `variant="subheading"` and `weight="bold"` before this? Same for `native.test.tsx`.

Reply:
What happens if I don't pass `weight="bold"`? What would be the default weight for a `subheading` variant? I can't see any weight being assigned when variant is subheading in the code. 

`h1-h6` tags are bold by default but what about for native? Won't we need to explicitly make it bold by default?